build-scripts/patch-libs-paths.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euvx
+
+# This script patches our NSS .dylibs to load each other using @rpath instead of
+# @executable_path which is incorrect.
```

Comment:
Is this incorrect because of some upstream bug in the NSS build process?

Reply:
Yes, technically it's possible to tell `clang` to change the id/rpath of dependant libraries, but it's a bit harder here since it's invoked through Makefiles for multiple libraries. It's just easier to fix it later with Apple's provided tool.